utils/utils_test.go

```diff
@@ -258,6 +258,162 @@ func TestCustomizeEnvSSO(t *testing.T) {
 	}
 }
 
+type licenseTestData struct {
+	// input from the spec
+	metric  webspherelibertyv1.LicenseMetric
+	edition webspherelibertyv1.LicenseEdition
+	pes     webspherelibertyv1.LicenseEntitlement
+	// whether CustomizeLicenseAnnotations is expected to return an err or not
+	pass bool
+}
+
+func TestCustomizeLicenseAnnotations(t *testing.T) {
+	t.Log("Starting license test")
+
+	td := []licenseTestData{}
+	td = append(td, licenseTestData{edition: webspherelibertyv1.LicenseEditionBase, pes: webspherelibertyv1.LicenseEntitlementStandalone})
+	td = append(td, licenseTestData{edition: webspherelibertyv1.LicenseEditionBase, pes: webspherelibertyv1.LicenseEntitlementCP4Apps})
+	td = append(td, licenseTestData{edition: webspherelibertyv1.LicenseEditionBase, pes: webspherelibertyv1.LicenseEntitlementFamilyEdition})
+	td = append(td, licenseTestData{edition: webspherelibertyv1.LicenseEditionBase, pes: webspherelibertyv1.LicenseEntitlementWSHE})
+
+	td = append(td, licenseTestData{edition: webspherelibertyv1.LicenseEditionBase, pes: webspherelibertyv1.LicenseEntitlementStandalone})
```

Comment:
This duplicate set (as a result of removing metrics) can be removed. Same for the other editions below.